lib/indico.rb

```diff
@@ -255,5 +255,49 @@ def clear(config = nil)
         config[:collection] = @collection
         Indico.api_handler(nil, 'custom', config, 'clear_collection')
       end
+
+      def rename(name, config = nil)
+        if config.nil?
```

Comment:
We should init Collections with a @config that contains the domain if not null & name, like we do in the other libs.